### PR TITLE
Delete concert contacts

### LINKED ISSUE #62: Delete contacts from concerts

### PR DESCRIPTION
**Pull request by hj235**
Fixes #62

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeletePersonCommand.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 import seedu.address.model.person.Person;
 
 /**
- * Deletes a person identified using it's displayed index from the address book.
+ * Deletes a person identified using its displayed index from the address book.
```
Comment (wes-w-z-h) [inline src/main/java/seedu/address/logic/commands/DeletePersonCommand.java:15]: it's is correct lol

File: src/main/java/seedu/address/logic/parser/DeleteConcertContactCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CONCERT;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.DeleteConcertContactCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new LinkCommand object
```
Comment (wes-w-z-h) [inline src/main/java/seedu/address/logic/parser/DeleteConcertContactCommandParser.java]: update command name

**Comment by hj235** [comment]
Haven't tested properly yet

File: src/main/java/seedu/address/logic/parser/DeleteConcertContactCommandParser.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+ */
+public class DeleteConcertContactCommandParser implements Parser<DeleteConcertContactCommand> {
+    /**
+     * Parses the given {@code String} of arguments in the context of the LinkCommand
+     * and returns an LinkCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
```
Comment (wes-w-z-h) [inline src/main/java/seedu/address/logic/parser/DeleteConcertContactCommandParser.java:18]: same here

**Review by wes-w-z-h — CHANGES_REQUESTED** [review]

**Review by wes-w-z-h — APPROVED** [review]

**Pull request merged**